src/main/java/com/gpuopenanalytics/jenkins/remotedocker/config/UserConfigItem.java

```diff
@@ -59,7 +59,13 @@ public UserConfigItem(boolean custom,
 
     public boolean isExisting() {
         return !isCustom()
-                && !("jenkins".equals(username) || "root".equals(username));
+                && !("jenkins".equals(username)
+                || "root".equals(username)
+                || "current".equals(username));
+    }
+
+    public boolean isCurrentUser() {
+        return "current".equals(username);
```

Comment:
Shouldn't this have one more Boolean op? like `&& !custom`

That way for what you have in the XML below you'll never run into the unlikely corner-case that some one has a custom user with the username `current`

I'm sure with the other custom user stuff it relies on `custom = true` but just thinking... also could be off I haven't had lunch yet 🤣

Reply:
You're correct